lib/actions/FrontendAction.js

```diff
@@ -30,20 +35,26 @@ class FrontendAction {
 
   /**
    * Runs the frontend process.
-   * @param {string} The action to perform.
+   * @param {string} action The action to perform.
    * @param {Object} [options={}] The process options.
    */
   run (action, options = {}) {
     if (!UserSettings.getInstance().getSession().getToken()) {
       throw new Error('not logged in')
     }
     const dcClient = new DcHttpClient(UserSettings.getInstance())
+    const themeConfigWatcher = new ThemeConfigWatcher()
     const frontendSetup = new FrontendSetup(dcClient)
     switch (action) {
       default:
       case 'start': {
         const frontend = new FrontendProcess(options, frontendSetup)
         frontend.run()
+          .then(() => {
+            themeConfigWatcher.start(() => {
```

Comment:
The watcher will not be stopped when a SIGINT comes in so the process never stops